tasks/main-blivet.yml

```diff
@@ -101,15 +31,39 @@
     name: "{{ package_info.packages }}"
     state: present
 
-- name: manage the pools and volumes to match the specified state
-  blivet:
-    pools: "{{ _storage_pools }}"
-    volumes: "{{ _storage_volumes }}"
-    use_partitions: "{{ storage_use_partitions }}"
-    disklabel_type: "{{ storage_disklabel_type }}"
-    safe_mode: "{{ storage_safe_mode }}"
-    diskvolume_mkfs_option_map: "{{ __storage_blivet_diskvolume_mkfs_option_map|d(omit) }}"
-  register: blivet_output
+- command: 'systemctl list-units --all --plain --full --no-legend -t service'
```

Comment:
Can you use `service_facts` instead?  https://docs.ansible.com/ansible/2.9/modules/service_facts_module.html#service-facts-module

Reply:
Changed to use `service_facts` in latest version. Bonus fun ensued related to `json_query`.